.github/workflows/libcxx-check-generated-files.yml

```diff
@@ -9,16 +9,24 @@ permissions:
 
 jobs:
   check_generated_files:
-    runs-on: ubuntu-latest
+    runs-on:
```

Comment:
I think there's a bit of confusion here. This job doesn't actually check for clang-format anymore. We should just stop installing clang-format in this job, I guess I missed that when we stopped enforcing clang-format via this job.

We exclusively enforce clang-format via the `pr-code-format.yml` workflow now.

Reply:
Yeah, I was trying to hijack it to test the added hooks, but it's trickier that I would have liked.